sycl/test-e2e/ESIMD/unified_memory_api/Inputs/atomic_update.hpp

```diff
@@ -341,8 +350,15 @@ bool test_fp_types_and_sizes(queue q, const Config &cfg) {
   passed &= test_fp_types<8, Op, UseMask, UsePVCFeatures>(q, cfg);
   // Supported by LSC atomic:
   if constexpr (UsePVCFeatures) {
-    passed &= test_fp_types<16, Op, UseMask, UsePVCFeatures>(q, cfg);
+    passed &= test_fp_types<12, Op, UseMask, UsePVCFeatures>(q, cfg);
```

Comment:
I think this 12 needs to be swapped with 16 on the line 359 below.

Reply:
Done.